dockerfiles/ubuntu/obam/README.md

```diff
@@ -11,6 +11,7 @@ This section defines the step-by-step instructions to build an [Ubuntu](https://
   > The hosted locations of artifacts will be passed as the build arguments when building the Docker image.<br>
   > 1. **WSO2_OB_Accelerator_DIST_URL** - Accelerator location
   > 2.  **WSO2_OB_CERTS_URL** - Certificate zip archive location
+  > 3.   **WSO2_OB_SERVER_CERTS_URL** - OBAM server certs folder location
```

Comment:
Looks like additional space is added at the beginning with each point. Lets remove those

Reply:
done [96b1427](https://github.com/wso2/docker-open-banking/pull/148/commits/96b1427a242688e58ad8fb681f691cc22c76f325)